lib/appSocketToClient.ts

```diff
@@ -24,23 +32,31 @@ export function connectToWebsocketServer(url = 'ws://websocket-server:3001') {
   socket.on('disconnect', () => {
     logger.info('Disconnected from Websocket');
   });
-  global.appSocketToWebSocketServer = socket as unknown as Socket;
+  global.appSocketToWebSocketServer[url] = socket as unknown as Socket;
 
-  return global.appSocketToWebSocketServer;
+  return global.appSocketToWebSocketServer[url];
 }
 
-export function requestBroadcastMatches() {
-  global.appSocketToWebSocketServer?.emit('broadcastMatches');
+export function requestBroadcastMatches(urls = WEBSOCKET_SERVER_URLS) {
+  for (const url of urls) {
+    global.appSocketToWebSocketServer[url]?.emit('broadcastMatches');
+  }
 }
 
-export function requestBroadcastMatch(matchId: string) {
-  global.appSocketToWebSocketServer?.emit('broadcastMatch', matchId);
+export function requestBroadcastMatch(matchId: string, urls = WEBSOCKET_SERVER_URLS) {
+  for (const url of urls) {
+    global.appSocketToWebSocketServer[url]?.emit('broadcastMatch', matchId);
+  }
 }
 
-export function requestScheduleBroadcastMatch(matchId: string) {
-  global.appSocketToWebSocketServer?.emit('scheduleBroadcastMatch', matchId);
+export function requestScheduleBroadcastMatch(matchId: string, urls = WEBSOCKET_SERVER_URLS) {
+  for (const url of urls) {
+    global.appSocketToWebSocketServer[url]?.emit('scheduleBroadcastMatch', matchId);
+  }
 }
 
-export function requestClearBroadcastMatchSchedule(matchId: string) {
-  global.appSocketToWebSocketServer?.emit('clearBroadcastMatchSchedule', matchId);
+export function requestClearBroadcastMatchSchedule(matchId: string, urls = WEBSOCKET_SERVER_URLS) {
+  for (const url of urls) {
```

Comment:
Why pass in urls? Why not just iterate thru the appSocletToWebSocket map?

Reply:
originally i was doing that but i changed it to get the tests to pass